### PR TITLE
improve contact menu, add about dialog to qt4

### PR DESCRIPTION
**Pull request by jamesaxl**
:)

### CODE REVIEW

File: emesene/gui/qt4ui/Dialog.py [hunk 1]
```diff
@@ -602,7 +636,10 @@ def accept_response(self):
         raise NotImplementedError()
 
     def reject_response(self):
-        raise NotImplementedError()
```
Comment (c10ud) [inline emesene/gui/qt4ui/Dialog.py]: here you're throwing an exception and catching it, if you want to "do nothing" just put pass

Comment (c10ud) [inline emesene/gui/qt4ui/Dialog.py]: (but you should override that method in your preferred dialog instead, so just leave this line as it is)

File: emesene/gui/gtkui/ContactMenu.py [hunk 2]
```diff
@@ -103,18 +103,23 @@ def __init__(self, handler, session):
...
+        nick_to_clipboard.set_image(gtk.image_new_from_pixbuf(nick_img))
 
         nick_to_clipboard.connect('activate',
             lambda *args: self.on_copy_nick_to_clipboard())
 
-        message_to_clipboard = gtk.MenuItem(_('Personal message'))
-
+        message_to_clipboard = gtk.ImageMenuItem(_('Personal message'))
+        message_to_clipboard.set_image(gtk.image_new_from_stock(gtk.STOCK_DIALOG_INFO,
+            gtk.ICON_SIZE_MENU))
```
Comment (c10ud) [inline emesene/gui/gtkui/ContactMenu.py]: isn't icon_size_menu 16x16? that's different from the 20x20 you're asking in other places

**Comment by c10ud** [comment]
Also for gtk: why aren't you using the utils module? it has utilities functions for loading images etc.

File: emesene/gui/qt4ui/Dialog.py [hunk 3]
```diff
@@ -240,6 +240,41 @@ def crop_image(cls, response_cb, filename, title=tr('Select image area')):
...
+        * version: version as string
+        * copyright: the name of the copyright holder
+        * comments: a description of the application
+        * license: the license text
+        * website: the website url
+        * authors: a list or tuple of strings containing the contributors
+        * translators: a string containing the translators
+        '''
+        image = gui.theme.image_theme.user
+        #QtGui.QMessageBox.about(None, 'About emesene',
```
Comment (Sbte) [inline emesene/gui/qt4ui/Dialog.py]: Useless comment?

**Pull request merged**